### PR TITLE
Serve group classifiers (Frontend)

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Provides frontend for #1719

image

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

**Comment by GitHK** [comment]
@odeimaiz how do I test this? Can't figure out there format of the entry in the database.

**Comment by GitHK** [comment]
> @odeimaiz how do I test this? Can't figure out there format of the entry in the database.

More on this. I manage to query via the API for the classifiers, but the frontend does no request to a /v0/groups/GID/classifiers endpoint. Is this correct?

**Comment by odeimaiz** [comment]
> > @odeimaiz how do I test this? Can't figure out there format of the entry in the database.
> 
> More on this. I manage to query via the API for the classifiers, but the frontend does no request to a **/v0/groups/{GID}/classifiers** endpoint. Is this correct?

Frontend does that call for all organizations you belong to. If you belong to 0 organizations, the frontend will make 0 calls

**Comment by GitHK** [comment]
> > > @odeimaiz how do I test this? Can't figure out there format of the entry in the database.
> > 
> > 
> > More on this. I manage to query via the API for the classifiers, but the frontend does no request to a **/v0/groups/{GID}/classifiers** endpoint. Is this correct?
> 
> Frontend makes that call for all organizations you belong to. If you belong to 0 organizations, the frontend will make 0 calls

I see now, will fix this

**Comment by GitHK** [comment]
> > > > @odeimaiz how do I test this? Can't figure out there format of the entry in the database.
> > > 
> > > 
> > > More on this. I manage to query via the API for the classifiers, but the frontend does no request to a **/v0/groups/{GID}/classifiers** endpoint. Is this correct?
> > 
> > 
> > Frontend makes that call for all organizations you belong to. If you belong to 0 organizations, the frontend will make 0 calls
> 
> I see now, will fix this

From what I can tell the gid in the `group_classifiers` table must be the `gid` of the `organization`!

**Review by GitHK — APPROVED** [review]
After the bundle is uploaded and associated with the correct `group gid` it works as expected.

File: services/web/client/source/class/osparc/utils/Classifiers.js [hunk 1]
```diff
@@ -25,47 +25,72 @@ qx.Class.define("osparc.utils.Classifiers", {
...
               };
-            }
-            return {
-              label: key.charAt(0).toUpperCase() + key.slice(1),
-              children: virtualTree(currentNode[key])
-            };
+              classifierPromises.push(osparc.data.Resources.get("classifiers", params));
+            });
+            Promise.all(classifierPromises)
+              .then(classifierss => {
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/utils/Classifiers.js:49]: MINOR: classifiersS<-- this is on purpose or a typo?

**Review by pcrespov — APPROVED** [review]

File: services/web/client/source/class/osparc/utils/Classifiers.js (hunk 1 above)
Comment (odeimaiz) [reply services/web/client/source/class/osparc/utils/Classifiers.js:49]: On purpose. classifierss: list of classifiers

**Pull request merged**